fbsimctl/FBSimulatorControlKit/Sources/CommandParsers.swift

```diff
@@ -72,6 +72,36 @@ extension Parser {
   }
 }
 
+extension Configuration : Parsable {
+  public static func parser() -> Parser<Configuration> {
+    return Parser
+      .ofTwoSequenced(
+        self.optionsParser(),
+        self.controlConfigurationParser()
+      )
+      .fmap { (options, controlConfiguration) in
+        return Configuration(controlConfiguration: controlConfiguration, options: options)
+      }
+  }
+
+  public static func controlConfigurationParser() -> Parser<FBSimulatorControlConfiguration> {
+    return Parser.ofTwoSequenced(
```

Comment:
It was in a few other places as well. E.g. `Parser<FBSimulatorManagementOptions>.unionOptions([
` and `Parser.ofTwo...`. Not sure if you want it to be consistent across the codebase.